__tests__/app.test.js

```diff
@@ -83,23 +102,56 @@ describe("API: /api/reviews", () => {
           expect(review.review_id).toBe(1);
         });
     });
+    test("201: responds with a votes decrementing by inc_vote", () => {
+      const review_id = 1;
+      const inc_vote = { inc_votes: -100 };
+
+      return request(app)
+        .patch(`/api/reviews/${review_id}`)
+        .send(inc_vote)
+        .expect(201)
+        .then(({ body: { review } }) => {
+          expect(review).toBeInstanceOf(Object);
+          expect(review.votes).toBe(-99);
+          expect(review.review_id).toBe(1);
+        });
+    });
   });
 
-  describe("errors: /api/reviews/:review_id", () => {
-    test("400: responds with an error message when passed an endpoint with an incorrect data type", () => {
-      const review_id = "invalid_type";
+  describe("PATCH - errors: /api/reviews/:review_id", () => {
+    test.only("400: responds with an error message when passed an endpoint with an incorrect data type", () => {
+      const review_id = 1;
+      const inc_vote = { inc_votes: "NaN" };
+
       return request(app)
-        .get(`/api/reviews/${review_id}`)
+        .patch(`/api/reviews/${review_id}`)
         .expect(400)
+        .send(inc_vote)
         .then(({ body: { message } }) => {
           expect(message).toBe("id is not valid");
```

Comment:
This message, doesnt match whats caused the error. They havent passed an id, they've passed not a number for the inc votes